docs/site/updating.md

```diff
@@ -1,32 +1,33 @@
 # Updating the Site
+
 The DMOJ is under active development, so occasionally you may wish to update. This is a fairly simple process.
 
-!>  The DMOJ development team makes no commitment to backwards compatibility. It's possible that an update migration
+!>  The DMOJ development team makes no commitment to backward compatibility. An update migration
```

Comment:
Both `backward` and `backwards` appear to be used, however, in contexts similar to this one `backwards` seems to be more used.

Reply:
`backwards` is more common.